### DISCUSSION TITLE
交互行为：可以触发触摸事件吗？

### DISCUSSION THREAD

**Discussion by BUPTlhuanyu** [galacean/effects-runtime#718] [Ideas]
只支持点击吗？web 上触摸事件可以拿到吗？

**Comment by yiiqii** [comment]
> 只支持点击吗？web 上触摸事件可以拿到吗？

你指的如果是 touchstart/move/end，暂时是未暴露的，如果你想实现类似拖拽的效果，现在可以直接用：https://galacean.antgroup.com/effects/user/cbnow0#fsGOm
